### PR TITLE
Issue #13345: Enabled TodoCommentCheckExamplesTest

### PR DESCRIPTION
**Pull request by MANISH-K-07**
Part of #13345

Enabled TodoCommentCheckExamplesTest

### CODE REVIEW

**Comment by MANISH-K-07** [comment]
GitHub, generate website

**Comment by MANISH-K-07** [comment]
@Vyom-Yadav , ping

**Review by rdiachenko — APPROVED** [review]
lgtm

**Comment by rdiachenko** [comment]
rebased on latest master

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/todocomment/Example1.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.todocomment;
+
+// xdoc section -- start
+public class Example1 {
+  int i;
+  public void test() {
+    i++; // TODO: do differently in future  // violation
+    i++; // todo: do differently in future
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/todocomment/Example1.java:15]: please do examples to be the same and only comments are changing.
it is ok to merge examples together to archive required result.

**Review by romani — CHANGES_REQUESTED** [review]
items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/todocomment/Example1.java (hunk 1 above)
Comment (MANISH-K-07) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/todocomment/Example1.java]: Done....
👍🏻

**Comment by MANISH-K-07** [comment]
GitHub, generate website

**Comment by romani** [comment]
GitHub, generate website

**Review by romani — APPROVED** [review]
ok to merge, thanks a lot!!!

**Pull request merged**